server/api.js

```diff
@@ -97,4 +98,55 @@ router.get("/modules", async (req, res) => {
 		console.log(error);
 	}
 });
+router.get("/feedback/student/:id", async (req, res) => {
+	try {
+		const userId = parseInt(req.params.id);
+		let modules = await db.query(
+			"SELECT * FROM users u INNER JOIN student_feedback sf ON u.user_id = sf.student_id INNER JOIN modules m ON sf.module_id = m.module_id WHERE u.user_id = $1 ORDER BY sf.date DESC",
+			[userId]
+		);
+		return res.json(modules.rows[0]);
+	} catch (error) {
+		console.log(error);
+	}
+});
+router.post("/feedback/student", auth, async (req, res) => {
```

Comment:
Is there a reason you added auth to this? because it disrupts the system. Can you re-check to see if the system does not break?

Reply:
I forgot to add the header on the front end, I'll do it

Reply:
Fixed